azure-units.py

```diff
@@ -347,12 +379,13 @@ def count_load_balancers(self):
         print('getting data for count_load_balancers')
         success, output = call_with_output(f"az network lb list {self.subscription_flag} --output json")
         if not success:
-            raise Exception("Failed with error: ",output)
+            print("Failed with error: ",output)
+            return 0
```

Comment:
returning 0 will give false notion that customer doesn't have any resource. Ideally we should send exception attached to the corresponding resource type so that it's evident we were not able to estimate it correctly. Logging using print might be missed.

Reply:
Created a new error column in the generated csv for clear understanding of failure case senario